packages/common/src/log.ts

```diff
@@ -0,0 +1,6 @@
+import { getModule } from './modules/container';
+import LogService from './services/logging/LogService';
+
+export const getLogger = () => {
```

Comment:
An alternative would be to add the 'scope' to the getLogger and store a single instance per use case instead of copying and pasting scope and getLogger.

Reply:
Hmm, interesting. It is not meant to be extended but acts as a shortcut to the LogService. If we add all methods to the log.ts (Logger.ts) file, we don't need the LogService anymore. The Logger.ts can then just call the method on transporters directly. What do you think?

Reply:
Ya, that's also probably fine. The only reason we have the log service is in case someone wants to override it at a higher level than the transporters, but the transporters look like they give us pretty good flexibility already, so we can probably collapse the hierarchy 1 level.

Reply:
Updated the PR